### PR TITLE
feature(TextEditor): UXD-1037 adding TextEditor.Tiny support

### PR DESCRIPTION
**Pull request by nahumzs**
### Purpose 🚀
Adding TextEditor ability to extend the init configuration properties in TinyMCE

```jsx
  <TextEditor isDisabled defaultValue={'<p>This is a text editor</p>'} apiKey={API_KEY}>
    <TextEditor.Tiny init={{ menubar: true }} />
  </TextEditor>
```

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/TextEditor/src/components/Tiny/Tiny.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export default function Tiny(props) {
+  // shell component
+  return <></>;
+}
+
+Tiny.displayName = "TextEditor.Tiny";
+
+Tiny.propTypes = {
+  /** this object can be any of the configuration options support by TinyMCE https://www.tiny.cloud/docs/configure/  */
+  init: PropTypes.shape({}).isRequired,
```
Comment (allison-c) [inline packages/TextEditor/src/components/Tiny/Tiny.js:14]: Why this wrapper is necessary? Why not just move it to `TextEditor ` ?

Comment (nahumzs) [reply packages/TextEditor/src/components/Tiny/Tiny.js:14]: to keep the way other components behave when modifying third-party libraries in our library, this way helps to bring clarity between paprika component and the tinyMCE props.

**Review by AndreyChernykh — APPROVED** [review]
LGTM

**Comment by allison-c** [comment]
No bump required?

**Comment by nahumzs** [comment]
> No bump required?

no, this component hasn't been released yet, I have started this process as a progressive PRs ...

**Pull request merged**